RInAppMessaging/Classes/Enums/Public/Identification.swift

```diff
@@ -1,5 +1,5 @@
-@objc public enum Identification: Int, Codable {
+internal enum Identification: Int, Codable {
```

Comment:
🎉 
we'll need a major semver update if we're breaking API won't we?

Reply:
Perhaps this change should be deferred until we are doing other important tasks that require a major semver release?

Reply:
That's probably a wise move 👍

Reply:
Ok, please remove and then add to the tech debt record page.

Reply:
Done